env.template

```diff
@@ -1,8 +1,7 @@
 # Copy this file to ".env.dev" and tweak values for local development
 FREQUENCY_URL=ws://0.0.0.0:9944
-REDIS_URL=redis://localhost
-REDIS_PORT=6379
 PROVIDER_ID=1
 PROVIDER_BASE_URL=https://some-provider
+REDIS_URL=redis://0.0.0.0:6389
```

Comment:
Should this line match up the ports definition in the docker-compose yaml?
`REDIS_URL=redis://0.0.0.0:6380`